### PR TITLE
Feat: Add cloud plugin cost management to admin section

### PR DESCRIPTION
**Pull request by abannachGrafana**
**What is this feature?**

- Add cost management UI plugin to admin section
- nest adaptive metrics and log volume explorer under cost management if feature toggle enabled

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: pkg/services/ngalert/migration/permissions_test.go
Comment (abannachGrafana) [inline pkg/services/ngalert/migration/permissions_test.go:579]: backend format commit hook made this change.

**Review by ashharrison90 — APPROVED** [review]
lgtm 👍 

probably want to wait for a review from a backender as well 😄

File: pkg/services/featuremgmt/registry.go
Comment (mmandrus) [inline pkg/services/featuremgmt/registry.go:902]: nit: remove the word "new"

**Review by mmandrus — APPROVED** [review]
LGTM with one small nitpick

**Comment by itsrohitsaroha** [comment]
@abannachGrafana with this feature available, can I use it the grafana opensource version which I am using on-prem server or this is only available for grafana cloud?

**Comment by abannachGrafana** [comment]
@itsrohitsaroha, It is my understanding that Cost Management, Adaptive Metrics, and Log Volume Explorer are cloud-only and don't have plans to be available for open-source consumption. @09jvilla may be able to answer more definitively.

**Comment by 09jvilla** [comment]
> It is my understanding that Cost Management, Adaptive Metrics, and Log Volume Explorer are cloud-only and don't have plans to be available for open-source consumption

Adam is correct. These are features specifically for our Cloud offering (available to all tiers, including both free and paying Grafana Cloud users).

**Pull request merged**
